tests/torch/test_sota_checkpoints.py

```diff
@@ -9,684 +9,535 @@
+    status: Optional[str] = None
+
+    def to_dict(self):
+        return {
+            "Model": self.model_name,
+            "Backend": self.backend,
+            "Metrics type": self.metric_type,
+            "Expected": self.expected,
+            "Measured": self.measured,
+            "Diff FP32": self.diff_fp32,
+            "Diff target": self.diff_target,
+            "Status": self.status,
+            "Build url": os.environ.get("BUILD_URL", ""),
+        }
+
+
+TEST_RESULT: List[ResultInfo] = []
+
+
+def add_test_result(run_info: ResultInfo):
+    TEST_RESULT.append(run_info)
+
+
+def read_reference_file(ref_path: Path) -> List[EvalRunParamsStruct]:
+    with ref_path.open(encoding="UTF-8") as source:
+        sota_eval_config = json.load(source, object_pairs_hook=OrderedDict)
 
-
-@pytest.mark.nightly
-class TestSotaCheckpoints:
     param_list = []
-    train_param_list = []
-    ids_list = []
-    train_ids_list = []
-    row_dict = OrderedDict()
-    color_dict = OrderedDict()
-    ref_fp32_dict = OrderedDict()
-    test = None
-
-    @staticmethod
-    def get_metric_file_name(model_name: str):
-        return "{}.metrics.json".format(model_name)
+    for sample_type_ in sota_eval_config:
+        datasets = sota_eval_config[sample_type_]
+        for dataset_name in datasets:
+            model_dict = datasets[dataset_name]
+            for model_name, sample_dict in model_dict.items():
+                if "target_pt" not in sample_dict:
+                    continue
+                param_list.append(
+                    EvalRunParamsStruct(
+                        model_name=model_name,
+                        config_name=sample_dict["config"],
+                        reference=sample_dict.get("reference", None),
+                        target_pt=sample_dict["target_pt"],
+                        target_ov=sample_dict["target_ov"],
+                        metric_type=sample_dict["metric_type"],
+                        dataset_name=dataset_name,
+                        sample_type=sample_type_,
+                        resume_file=sample_dict.get("resume", None),
+                        batch=sample_dict.get("batch", None),
+                        diff_fp32_min=sample_dict.get("diff_fp32_min", DIFF_FP32_MIN_GLOBAL),
+                        diff_fp32_max=sample_dict.get("diff_fp32_max", DIFF_FP32_MAX_GLOBAL),
+                        diff_target_ov_min=sample_dict.get("diff_target_ov_min", DIFF_TARGET_OV_MIN),
+                        diff_target_ov_max=sample_dict.get("diff_target_ov_max", DIFF_TARGET_OV_MAX),
+                        diff_target_pt_min=sample_dict.get("diff_target_pt_min", DIFF_TARGET_PT_MIN),
+                        diff_target_pt_max=sample_dict.get("diff_target_pt_max", DIFF_TARGET_PT_MAX),
+                        multiprocessing_distributed=sample_dict.get("multiprocessing_distributed", False),
+                        skip_ov=sample_dict.get("skip_ov", ""),
```

Comment:
The default value should probably be `None` or `False`

Reply:
Fixed